caluma/caluma_form/serializers.py

```diff
@@ -625,6 +585,7 @@ class Meta:
 class RemoveDocumentSerializer(serializers.ModelSerializer):
     document = serializers.GlobalIDField(source="id")
 
+    @transaction.atomic
     def update(self, instance, validated_data):
         if hasattr(instance, "case"):
             raise Exception("You cannot remove a Document, if it's attached to a case.")
```

Comment:
Can you remove a document if it's family is attached to a case?

Reply:
I don't understand your question. I've only added the atomic transaction for consistency, didn't touch the mutation otherwise.

Reply:
The question wasn't related to the PR it just popped up when I read it. NVM